Add Web export file name section (#10142)
godotengine/godot-docs: tutorials/export/exporting_for_web.rst

    general, especially when using the GLES2 rendering backend (which only
    requires WebGL 1.0).

Export file name
----------------

We do suggest users to export their Web projects with ``index.html`` as the file name.

> ``index.html`` is usually interpreted as the default file to load by Web servers when
> accessing the parent directory, and usually hide the name of that file.

Reviewer: ```suggestion
``index.html`` is usually the default file loaded by web servers when
accessing the parent directory, usually hiding the name of that file.
```

Author: Done!
